### PR TITLE
added steps to add a new service in RHTAP

### PR DESCRIPTION
**Pull request by manish-jangra**

### CODE REVIEW

File: docs/deployment/extending-the-service.md [hunk 1]
```diff
@@ -12,24 +12,105 @@ You may use the [gitops](../../components/gitops/) component as an example for h
...
+>
+4. Add a reference to your new `(team-name).yaml` file, to `argo-cd-apps/base/directory/team-name/kustomization.yaml` or `argo-cd-apps/base/team-name/kustomization.yaml` (the reference to your YAML file should be in the `resources:` list field).
+
+    ```YAML
+    apiVersion: kustomize.config.k8s.io/v1beta1
+    kind: Kustomization
+    resources:
+    - team-name.yaml
+    ```
+5. Kustomize Components a new kind of Kustomization that allows users to define reusable kustomizations. Components can be included from higher-level overlays to create variants of an application, with a subset of its features enabled. We have `argo-cd-apps/k-components` directory in this reposiroty to plcae `kustomization.yaml` with `kind: Component`.
```
Comment (bamachrn) [inline docs/deployment/extending-the-service.md:81]: typo plcae -> place

**Review by bamachrn — COMMENTED** [review]
looks good otherwise

**Comment by manish-jangra** [comment]
The typo has been fixed.

**Review by bamachrn — APPROVED** [review]
/lgtm

**Pull request merged**